Finjector.Web/ClientApp/src/pages/Entry.tsx

```diff
@@ -80,7 +83,12 @@ const Entry = () => {
 
   return (
     <div className="main">
-      <h1>Create new Chart</h1>
+      <a className="back-link" href="/landing">
+        <FontAwesomeIcon icon={faArrowLeft} />
+        Back
+      </a>
+
+      <h2>Chart Type</h2>
```

Comment:
Can't use <a> in a react app, unless it's to an external site.  need to change to <Link /> from the react router lib